### ISSUE TITLE
Window freezes after scanning QR code

### ISSUE THREAD

**Issue by hqurve** [eneshecan/whatsapp-for-linux#120]
**Description**
Scanning the QR code to log into web.whatsapp.com causes the window to freeze and draw events are no longer processed (ie resizing the window results in a blank screen).

**To Reproduce**
1. Launch application
2. Scan the QR code to log in

**Screenshots**
![Screenshot from 2021-06-10 20-24-31](https://user-images.githubusercontent.com/53281855/121613146-e0dd4700-ca29-11eb-9747-30289d4822d2.png) 

**Environment**
 - Distro nixos
 - Package whatsapp-for-linux
 - Desktop Environment or Window Manager: kde plasma 5.21.5

**Comment by hqurve** [comment]
I also downloaded the nix file and updated the version to 1.17 and the issue persisted. Additionally, upon changing the URI to google.com, the application worked well (including going on other websites). I dont believe that the issue is nixos packaging but either the application or the website.

**Comment by eneshecan** [comment]
Hi @hqurve, thanks for reporting. I couldn't reproduce the problem, could you please check application outputs by starting from the terminal?

**Comment by hqurve** [comment]
Hi, I just rebuilt off master and the console shows no errors, whilst still freezing. Is there a way to send the browser's console to stdout or make it visible?

Also, for testing/fun, I decided to set the URI to google.com. I then navigated to the snake game, it worked well (albeit with several initial glib errors). Finally, I went to whatsapp web and the qrcode came up, but as before, when I scanned the qr code, the window froze.
